src/Form/ServiceSettingsForm.php

```diff
@@ -91,7 +90,8 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
           $form_state->getValue('bpi_api_key'),
           $form_state->getValue('bpi_secret_key')
         );
-      } catch (SDKException $e) {
+      }
+      catch (SDKException $e) {
```

Comment:
Rather than just logging any exception, you should also inform the user (via `setErrorByName`, say) that the connection cannot be established.

Reply:
Yes: https://github.com/easySuite/bpi/pull/6/files#diff-ae10fa67a401e5d93da8e99eb57d35c0R95